lib/pbench/test/functional/server/test_connect.py

```diff
@@ -29,5 +29,5 @@ def test_connect(self, server_client: PbenchServerClient):
 
         # verify all the required openid-connect fields are present
         if "openid" in endpoints:
-            expected = {"server", "client", "realm"}
-            assert set(endpoints["openid"]) == expected
+            expected = {"client", "realm", "server"}
```

Comment:
This is a `set`, so I don't think the order actually matters. Doesn't hurt to put it in sorted, though.